to-do-list/src/API/PostService.tsx

```diff
@@ -0,0 +1,39 @@
+import axios from 'axios'
+import {PostItemProps} from '../components/Interfaces/interfaces'
+
+export default class PostService {
+    static async getAll(){
+        try{
+            const response = await axios.get('db.json')
+            return response.data
+        } catch(e){
+            console.log(e)
```

Comment:
don't handle errors with console.log. User won't see these messages, so it's better to extract try/catch to the upper level and create a toaster component on error to notify the user